worker-sandbox/src/router.rs

```diff
@@ -452,6 +452,18 @@ pub fn make_router<'a>(data: SomeSharedData) -> Router<'a, SomeSharedData> {
             "/durable/auto-response",
             handler!(crate::test::auto_response::handle_auto_response),
         )
+        .get_async(
```

Comment:
I realized I didn't add the corresponding routes when the `http` feature is enabled. Addressed in https://github.com/cloudflare/workers-rs/pull/763.